### ISSUE TITLE
Bot API Endpoints

### ISSUE THREAD

**Issue by aurium** [actorapp/actor-platform#184]
Hi, i'll not ask for a Telegram cloned API and full docs, but, someone could point me how to connect some bot API endpoints, considering i'm not using Java? For now I only want to know the urls to get user messages and the url to send messages.

Thanks!

**Comment by ex3ndr** [comment]
You can use web hooks: https://actor.readme.io/docs/simple-integration

**Comment by aurium** [comment]
Yeah... but that demands a machine with public ip. That makes hard to dev/test in a LAN. The java module uses some other endpoints... Would be nice to find some doc.
However, thank you!
